tests/materials/processing/models/test_material.py

```diff
@@ -0,0 +1,63 @@
+from slamd.materials.processing.models.additional_property import AdditionalProperty
+from slamd.materials.processing.models.material import Material, Costs
+
+
+def test_material_constructor_sets_default_values():
+    material = Material()
+
+    assert material.name == ''
+    assert material.type == ''
+    assert material.costs == None
+    assert material.additional_properties == None
+    assert material.is_blended == False
```

Comment:
for consistency we should also use the "is" operator in boolean assertions, i.e. is True instead of == True (similarly for False). Note that this comment applies to all test which use the equality operator.